Source/IntegrationTests/TestFiles/LitTests/LitTest/gomodule/dependency-module/DerivedModule.dfy

```diff
@@ -0,0 +1,13 @@
+// RUN: %baredafny translate go  --module-name=GoModule2 --library="%S/test.doo" --translation-record "%S/test-go.dtr" --output "%S/test3" "%s"
+// RUN: cp -r "%S/go.*" "%S/test3-go/"
+// RUN: cp -r "%S/../module-libraries/DafnyRuntimeGo" "%S"
+// RUN: cp -r "%S/DafnyModule1" "%S/test3-go/"
```

Comment:
I'm not clear on why moving the files around so much is necessary (probably because I don't understand Go as well :)

Reply:
This is the side effect of how the test framework is implemented, how compiler skips generating go.mod, how we don't have DafnyRuntimeGo published as module, and how go needs everything in a directory. Almost in that order.